engine/apps/api/serializers/user.py

```diff
@@ -324,6 +329,10 @@ class Meta:
             "avatar_full",
         ]
 
+    def get_avatar_full(self, obj):
```

Comment:
Is there any way to reuse the implementation above? (to avoid the duplication)

Reply:
We have so many user serializers that inherit from each other, so I think it would be better to reuse this since it's just a couple of lines to avoid additional overcomplication 🤔 wdyt?

Reply:
Sounds ok, I guess eventually we may want to define a mixin class that it can be inherited to provide the functionality if there is more shared bits.